pkg/vcs/git.go

```diff
@@ -635,3 +635,26 @@ func (git *git) CommitExists(commit string) (bool, error) {
 	}
 	return true, nil
 }
+
+func (git *git) PushCurrentCommitAsTagTo(repo string, gitConfig ...string) error {
+	if repo == "" {
+		return nil
+	}
+	if len(gitConfig) != 0 {
+		gitConfigParams := append([]string{"config"}, gitConfig...)
+		if _, err := git.git(gitConfigParams...); err != nil {
+			return fmt.Errorf("git %s: %w", strings.Join(gitConfigParams, " "), err)
+		}
+	}
+	commit, err := git.HeadCommit()
+	if err != nil {
+		return fmt.Errorf("git.HeadCommit: %w", err)
+	}
+	tagName := commit.Hash
```

Comment:
Is it not too ambiguous if we start using tag names equal to the commit hashes they point to? Maybe it'd be better to use some tag name that includes e.g. the syz-manager's name, the current date and a prefix of the commit hash? It will be much more informative and human friendly and there will be no confusion at all.

Reply:
It is a commit->data cache. I don't expect us to use it differently.
With high probability commit is enough to get other data from the datastore "build" objects.

Reply:
I mean, inside that repository it will be quite confusing to see the same names being both a tag and a hash. Let's at least do something like `tagName := fmt.Sprintf("archive-%s", commit.Hash)` to differentiate them.

Reply:
Let's "tag-" + commit.Hash.